web/themes/new_weather_theme/templates/block/block--weathergov-daily-forecast.html.twig

```diff
@@ -1,20 +1,23 @@
 <div class="daily-forecast-block">
   {{ attach_library('new_weather_theme/hourly-toggle') }}
   {% if content.error %}
-  {% set message = "There was an error loading the daily forecast." | t %}
+    {% set message = "There was an error loading the daily forecast." | t %}
   {% include '@new_weather_theme/partials/uswds-alert.html.twig' with { 'level': "error", body: message } %}
   {% else %}
 
-  <div class="grid-container padding-x-0 tablet:padding-x-2">
-    <ol class="usa-list--unstyled">
+    <div class="grid-container padding-x-0 tablet:padding-x-2">
+      <ol class="usa-list--unstyled">
       {# Attempt to render the period(s) associated with the current day #}
-      {% if content.today %}
-      {% include '@new_weather_theme/partials/daily-forecast-list-item.html.twig' with { 'periods': content.today, 'dayHours': content.todayHourly, 'alertPeriods': content.todayAlerts, 'dayLabel': "Today" | t } %}
+        {% if content.today %}
+          {% set precipHours = content.precipitationPeriods[0] %}
+          {% include '@new_weather_theme/partials/daily-forecast-list-item.html.twig' with { 'periods': content.today, 'dayHours': content.todayHourly, 'alertPeriods': content.todayAlerts, 'precipHours': precipHours, 'dayLabel': "Today" | t } %}
+          {% set precipIndex = 1 %}
```

Comment:
I'll fetch this locally and play with it tomorrow, but looking at it tonight, I can't see where `precipIndex` ever gets incremented.

Reply:
line 19. We add the index of the current loop to the precipIndex. Probably poorly named, should be "offset" or something like that

Reply:
Ooooh, I see now. I was expected `precipIndex` to get incremented over time, but that's not its purpose. It's there to advance the index past the "today" precip table while iterating through the "detailed" periods. Got it now!